src/models.rs

```diff
@@ -205,6 +226,28 @@ impl TimeEntry {
             format!("[{}]", self.tags.join(", "))
         }
     }
+
+    pub fn as_network_time_entry(&self) -> NetworkTimeEntry {
```

Comment:
Might be neat to convert these into rust's conventional `into` methods

https://doc.rust-lang.org/std/convert/trait.Into.html